src/cargo/core/compiler/output_sbom.rs

```diff
@@ -0,0 +1,241 @@
+//! cargo-sbom precursor files for external tools to create SBOM files from.
+//! See [`output_sbom`] for more.
+
+use std::{
```

Comment:
nit: in Cargo we don't do nested multi-line import, though this is not enforced :)